.github/workflows/command-l10n-update.yml

```diff
@@ -50,21 +50,10 @@ jobs:
           git config --local user.email "nextcloud-command@users.noreply.github.com"
           git config --local user.name "nextcloud-command"
 
-      - name: Read package.json node and npm engines version
-        uses: skjnldsv/read-package-engines-version-actions@v3
-        id: package-engines-versions
+      - name: Set up node
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v3
         with:
-          fallbackNode: '^20'
-          fallbackNpm: '^10'
-
-      - name: Set up node ${{ steps.package-engines-versions.outputs.nodeVersion }}
-        uses: actions/setup-node@v4
-        with:
-          node-version: ${{ steps.package-engines-versions.outputs.nodeVersion }}
-          cache: npm
-
-      - name: Set up npm ${{ steps.package-engines-versions.outputs.npmVersion }}
-        run: npm i -g npm@"${{ steps.package-engines-versions.outputs.npmVersion }}"
+          node-version-file: package.json
```

Comment:
But it doesn't support `devEngines` yet, right?

Reply:
https://github.com/actions/setup-node/issues/1255

Reply:
So no but works as long as devEngines are in the upper bound of the engines.
IMHO browser libraries like this do not even need any `engine` as they only target the browser not Node (in contrast to e.g. vite-config which targets Node).